atom/browser/native_window.cc

```diff
@@ -517,6 +517,10 @@ void NativeWindow::NotifyWindowLeaveFullScreen() {
                     OnWindowLeaveFullScreen());
 }
 
+void NativeWindow::NotifyDevToolsFocused() {
```

Comment:
We don't have to add a `NotifyDevToolsFocused` method if there is only one place to call `OnDevToolsFocus` and it is in `NativeWindow`.